src/app/Unauthenticated.tsx

```diff
@@ -1,53 +1,18 @@
-import { Card, CardContent, Grid, Typography } from '@mui/material';
-import { Auth } from '@supabase/ui';
-import Topbar from 'components/header/Topbar';
-import { FormattedMessage } from 'react-intl';
-import { supabase } from 'services/supabase';
-import { getLoginSettings } from 'utils/env-utils';
+import { Skeleton } from '@mui/material';
+import { lazy, Suspense } from 'react';
+import { Route, Routes } from 'react-router';
 
-const Unauthenticated = () => {
-    const loginSettings = getLoginSettings();
+const Login = lazy(() => import('../pages/Login'));
```

Comment:
random thought... should be lazy load login? That might be something that should always be available.